UPDATE.md

```diff
@@ -65,6 +67,10 @@ Beta 2, then from Beta 2 to Beta 3, in that order.
   actively using Page Manager, you must add it to your project as an explicit
   dependency in order to continue to using it. Otherwise, you should uninstall
   it before updating to the next version of Lightning.
+* Lightning now supports exposing all Drupal entities as JSON, in the standard
+  JSON API format. To enable this feature, install the OpenAPI, OpenAPI Redoc
+  Documentation, and JSON API modules. Then install the Content API module from
+  the Lightning package.
```

Comment:
Probably need to understand this before I can really comment on any of the reset.